### DISCUSSION TITLE
Apple iOS - Testflight

### DISCUSSION THREAD

**Discussion by JigSawFr** [astubenbord/paperless-mobile#100] [General]
Hi there,
Do you think that's possible for you to compile and send every release to Apple Testflight ? (with CI it should be possible)
https://www.cobeisfresh.com/blog/how-to-implement-a-ci-cd-workflow-for-ios-using-github-actions
Thank you ! and have a nice day.

**Comment by astubenbord** [comment]
Hi, wouldn't this require an apple developer account?

**Comment by JigSawFr** [comment]
> Hi, wouldn't this require an apple developer account?

Think so, yes

**Comment by astubenbord** [comment]
I see, since I primarily develop on Android and it has a fee of 100€/year which I am not willing to spend, I will probably stick to manual releases here on GitHub. These will not be as regular as for Android though as I'm not a Mac user, but I have access to a MacBook. I might build the current version for iOS and add it to the release.

Sorry!

**Comment by JigSawFr** [comment]
Np, think that you can use GitHub Sponsor to try covering fees.
On my side will try to fork and play with GH CI to release a TestFlight version.
Have a nice day

**Comment by astubenbord** [comment]
I'm not sure there are enough users interested in a regularly updated iOS version willing to help cover the fees. But I could start a sponsor campaign and see what happens ;)

**Comment by astubenbord** [comment]
And btw, once you got something set up feel free to hit me up, I'd be interested in the solution!

**Comment by astubenbord** [comment]
I've opened a discussion to find out whether this is something more people would be interested in (see #35)
